nitro-src/claim.ts

```diff
@@ -49,13 +89,13 @@ export function claim(
       destinationIndex < destinations.length;
       destinationIndex++
     ) {
-      if (surplus.isZero()) break;
+      if (surplus.lte(0)) break;
```

Comment:
same question: I think we know that surplus is nonzero? If it is negative, I think something has gone badly wrong already?

Reply:
4b75fd6